### PR TITLE
chore(bundle): Clean up dependencies, depend on types where possible

### PR DESCRIPTION
**Pull request by GordonSmith**
Bump Versions:
 * Lerna
 * CodeMirror
 * Dagre

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage this PR does a lot of the d3 de-duping you were looking for...

**Comment by jbrundage** [comment]
@GordonSmith I see a lot of `d3SelectionType` removal. Are we no longer going to assign that type to d3 selections?

File: packages/graph/src/GraphData.ts [hunk 1]
```diff
@@ -62,80 +99,56 @@ export class GraphData extends graphlib.Graph {
...
+    edges(): GraphEdge[] {
+        return this._g.edges().map(e => this._g.edge(e));
     }
 
-    neighbors(nodeID: string) {
-        return super.neighbors(nodeID)
-            .map(item => this.node(item));
+    neighbors(id: string) {
+        const d = this._g.neighbors(id);
+        console.log(d);
```
Comment (jbrundage) [inline packages/graph/src/GraphData.ts]: console.log

File: packages/graph/src/GraphData.ts [hunk 2]
```diff
@@ -62,80 +99,56 @@ export class GraphData extends graphlib.Graph {
...
+        return this._g.neighbors(id).map(n => this._g.node(n));
     }
 
-    singleNeighbors(nodeID: string) {
-        return super.neighbors(nodeID)
-            .filter(item => super.neighbors(item).length === 1)
-            .map(item => this.node(item));
+    singleNeighbors(id: string) {
+        const d = this._g.neighbors(id);
+        console.log(d);
```
Comment (jbrundage) [inline packages/graph/src/GraphData.ts]: console.log

Comment (jbrundage) [reply packages/graph/src/GraphData.ts]: Looks like the line about console.log can be removed too.

File: packages/graph/src/GraphData.ts (hunk 1 above)
Comment (jbrundage) [reply packages/graph/src/GraphData.ts]: Looks like the line about console.log can be removed too.

**Comment by jbrundage** [comment]
Awesome, I'm looking forward to seeing how much smaller the individual dist files will be after this is merged.

**Pull request merged**